FluentTerminal.App/Strings/de/Resources.resw

```diff
@@ -832,4 +832,8 @@ Fuzzy</comment>
   <data name="ConfirmDeleteProfile" xml:space="preserve">
     <value>Sind Sie sicher dass Sie dieses Profil löschen wollen?</value>
   </data>
+  <data name="SshProfiles.Text" xml:space="preserve">
```

Comment:
please do not add them manually (they will be overwritten), register as a contributor instead: https://poeditor.com/join/project/TfKNIt826z

Reply:
Ok, sorry.